data/World/Dodongos Cavern MQ.json

```diff
@@ -42,7 +42,61 @@
                 can_play(Song_of_Time) and (can_child_attack or is_adult)",
             "Dodongos Cavern MQ GS Larvae Room": "can_use(Sticks) or has_fire_source",
             "Dodongos Cavern MQ GS Lizalfos Room": "can_blast_or_smash",
-            "Dodongos Cavern MQ Deku Scrub Staircase": "can_stun_deku"
+            "Dodongos Cavern MQ Deku Scrub Staircase": "can_stun_deku",
+            "Dodongos Cavern MQ Torch Puzzle Room Recovery Heart": "can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire)",
+            "Dodongos Cavern MQ Staircase Pot 1": "True",
+            "Dodongos Cavern MQ Staircase Pot 2": "True",
+            "Dodongos Cavern MQ Staircase Pot 3": "True",
+            "Dodongos Cavern MQ Staircase Pot 4": "True",
+            "Dodongos Cavern MQ Upper Lizalfos Pot 1": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Upper Lizalfos Pot 2": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Upper Lizalfos Pot 3": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Upper Lizalfos Pot 4": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
+            "Dodongos Cavern MQ Torch Puzzle Room Pot 1": "
+                can_blast_or_smash or can_use(Sticks) or can_use(Dins_Fire) or 
+                (is_adult and (logic_dc_jump or Hover_Boots or Progressive_Hookshot))",
```

Comment:
Whichever of these two pots is the one on the pillar, it can be obtained with the Boomerang from below, just like in vanilla DC. However, if you have Boomerang, you can kill a Baba earlier in the dungeon and get a Stick, which means Boomeranging this pot in logic doesn't practically do anything except allow the playthrough to maybe skip needing sticks before getting this pot.